runtime/shell.go

```diff
@@ -72,6 +72,8 @@ func (shell *Shell) ReadSpecialVar(name string) string {
 		return strconv.FormatInt(int64(len(shell.Args)-1), 10)
 	case "?":
 		return strconv.FormatInt(int64(shell.ExitCode), 10)
+	case "*", "@":
+		return strings.Join(shell.Args[1:], " ")
```

Comment:
Please note that running `exec some.exe "$@"` is different from running `exec some.exe "$*"`. The latter combines all arguments `$1`, `$2`, etc into a single parameter by joining them with spaces. The former, however, passes the same parameters _individually_ to the executable.

Reply:
@yassinebenaid ☝️